odybcl2fastq/config.py

```diff
@@ -32,8 +32,11 @@ def keys(self):
     def __contains__(self, key):
         return key in self.data
 
-    def check_dir(self, path):
+    def check_dir(self, path, check_is_writable=False):
```

Comment:
check_dir is a local function in config so it only applies to what we intend to do here, do we ever want to check that the path exists but not want to check that it is writable?  If no then I would suggest removing the parameter and just checking for writability always.  For clarity you could add a comment for the function that states it does both, but not totally necessary.

Reply:
In their current usage, SOURCE_CLUSTER_PATH and REF_PATH are not written to, and so they needn't be writable.

Reply:
ok sounds reasonable.